Formula/ffmpeg.rb

```diff
@@ -7,6 +7,7 @@ class Ffmpeg < Formula
     sha256 "5ef5e9276c311c74ab2e9d301c2d7ee10e1f2cbd758c6f13d6cb9514dffbac7e"
 
     depends_on "yasm" => :build
+    option "with-schroedinger", "Enable Dirac video format"
```

Comment:
and actually, sorry my mistake, this should be above `depends_on "yasm" => :build`, with a blank line beneath it, and the `depends_on "schroedinger" => :optional` needs to move up here below `depends_on "yasm" => :build`.